lib/router/tasks.js

```diff
@@ -25,5 +25,14 @@ module.exports = (taskflow) => {
       .catch(next);
   });
 
+  router.get('/profile-tasks/:profileId', (req, res, next) => {
+    const profileId = req.params.profileId;
+    const send = taskflow.responder(req, res);
+    Promise.resolve()
+      .then(() => queryBuilder.getSubjectTasks(profileId))
+      .then(send)
+      .catch(next);
+  });
```

Comment:
This should be a separate router - like https://github.com/UKHomeOffice/asl-workflow/blob/master/lib/router/model-tasks.js - rather than being piggy-backed onto the global task lists.